hudi-common/src/main/java/org/apache/hudi/common/table/checkpoint/CheckpointV1.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.hudi.common.table.checkpoint;
+
+import org.apache.hudi.common.model.HoodieCommitMetadata;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CheckpointV1 extends Checkpoint {
+  // TODO(yihua): decouple the keys for Hudi Streamer
+  public static final String STREAMER_CHECKPOINT_KEY_V1 = "deltastreamer.checkpoint.key";
```

Comment:
not a fan of having streamer configs here in `hudi-common`.. Should we just have Checkpoint abstract class here.. and move others close to their own packages..

Reply:
rename: StreamerCheckpointV1.